qlib/config.py

```diff
@@ -92,6 +92,7 @@ def set_conf_from_C(self, config_c):
     "kernels": NUM_USABLE_CPU,
     # How many tasks belong to one process. Recommend 1 for high-frequency data and None for daily data.
     "maxtasksperchild": None,
+    "joblib_backend" : None,
```

Comment:
Can we set the default backend to multiprocessing if loky is very likely to OOM?